pkg/apis/batch/validation/validation.go

```diff
@@ -75,7 +75,7 @@ var (
 //
 // TODO: generalize for other controller objects that will follow the same pattern, such as ReplicaSet and DaemonSet, and
 // move to new location.  Replace batch.Job with an interface.
-func ValidateGeneratedSelector(obj *batch.Job) field.ErrorList {
+func ValidateGeneratedSelector(obj *batch.Job, validateBatchLabels bool) field.ErrorList {
```

Comment:
this probably doesn't need to be exported

Reply:
updated.